client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -125,7 +125,7 @@ const useStatusChangeButton = (requisition: ResponseFragment) => {
     if (!selectedOption) return null;
     let result;
     try {
-      result = await save({ id, status: selectedOption.value });
```

Comment:
Need to keep this bit! Otherwise aren't saving the status change 🙈

Reply:
Ah thanks for spotting :) Have reverted it now!